File: src/main/java/com/clearcont/clearcontapp/repository/EmpresaGroupRepository.java
package com.clearcont.clearcontapp.repository;

import com.clearcont.clearcontapp.model.EmpresaGroup;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface EmpresaGroupRepository extends JpaRepository<EmpresaGroup, Long> {
}

File: src/main/kotlin/br/com/clearcont/clearcontwebapp/repository/EmpresaGroupRepository.kt
package br.com.clearcont.clearcontwebapp.repository

import br.com.clearcont.clearcontwebapp.models.EmpresaGroup
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface EmpresaGroupRepository : JpaRepository<EmpresaGroup, Long>
